src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs

```diff
@@ -449,6 +460,12 @@ public static FormulaValue OptionSetValueToString(IRContext irContext, OptionSet
             var displayName = optionSet.DisplayName;
             return new StringValue(irContext, displayName);
         }
+
+        public static FormulaValue BooleanOptionSetToBoolean(IRContext irContext, OptionSetValue[] args)
+        {
+            var arg0 = args[0];
+            return arg0.Option == "1" ? FormulaValue.New(true) : FormulaValue.New(false);
```

Comment:
I wonder if we should throw / error if it's not "1" or "0".

Reply:
Good question. From DV we know we can expect "1" or "0". I'm wondering if we should check for "true"/"false".